src/components/Button.js

```diff
@@ -13,11 +13,11 @@ const MainNavButton = styled(Button)`
   padding: 5px;
   width: 75px;
   height: 60px;
-  position: fixed;
+  /* position: fixed;
   bottom: 0;
   left: 50%;
-  transform: translate(-50%);
-  z-index: 1;
+  transform: translate(-50%); */
+  /* z-index: 1; */
 `;
```

Comment:
are you not sure to use it again? otherwise you could delete the comments ;)

Reply:
Haha sorry, i thought i deleted it 🙈